feat: added `idProvider` to model constructor (#483)
blackflux/dy-alchemy: README.md

_Params_

> * `id` string: Id of model to create, must be unique. _Optional if primaryKeys configured_.

Reviewer: fix

> * `id` string: Id of model to create, must be unique. _Must provide id or configure primaryKeys but not both_.

Reviewer: ```suggestion
* `id` string: Id of model to create, must be unique. _Required iff primaryKeys no provided_.
```
